### PR TITLE
[Feature] Support top N entities and thermodynamic metrics

### PR DESCRIPTION
**Pull request by kezhenxu94**
### Motivation

Support top N entities and thermodynamic metrics commands

### Result

- Commands `metrics top n` and `thermodynamic` are added

- Closes https://github.com/apache/skywalking/issues/3898

- Closes https://github.com/apache/skywalking/issues/3897

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -395,6 +425,50 @@ $ ./bin/swctl-latest-darwin-amd64 --display=graph --debug metrics multiple-linea
...
+```
+
+</details>
+
+<details>
+
+<summary>Query the overall heatmap</summary>
+
+```shell
+$  ./bin/swctl-latest-darwin-amd64 metrics thermodynamic --name all_heatmap
```
Comment (arugal) [inline README.md]: is `swctl-latest-darwin-amd64`?

**Comment by wu-sheng** [comment]
Heatmap without graph?

**Comment by wu-sheng** [comment]
Is this intended?

**Comment by kezhenxu94** [comment]
> Heatmap without graph?

The Ascii Graph lib doesn't support heat map for now, we may need to implement ourselves, but it takes a lot of efforts, will consider in the future

**Pull request merged**